composer.json

```diff
@@ -6,8 +6,9 @@
     "license":     "MIT",
     "require": {
         "php": ">=5.4",
-        "nekland/base-api": "~1.1",
-        "guzzlehttp/guzzle": "~5.0"
+        "nekland/base-api": "~1.1.0",
+        "guzzlehttp/guzzle": "~5.0",
+        "symfony/symfony": "2.6.*"
```

Comment:
Don't add complete symfony framework but just symfony http fundation. And as the File class is here for a while, I think you can just specify `^2.3.0`. (composer understand it as "all versions before 3.0 accepted") 

https://github.com/symfony/http-foundation